app/services/timeline_events/create_service.rb

```diff
@@ -29,13 +29,12 @@ def update_latest_flag(timeline_event)
         .where(timeline_event_owners: { founder: @founder.startup.founders })
         .where.not(id: timeline_event)
 
-      old_events.where(latest: true).each do |submission|
-        next if submission.founder_ids != @founder.startup.founder_ids
-
-        submission.update!(latest: false)
-      end
+      # rubocop:disable Rails/SkipsModelValidations
+      TimelineEventOwner.where(timeline_event_id: old_events, founder: @founder.startup.founders).update_all(latest: false)
```

Comment:
This probably has the same issued I've detailed in my comment for `UndoSubmissionMutator`.